shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/database/model/TokenDatabaseModel.kt

```diff
@@ -0,0 +1,36 @@
+package co.nimblehq.blisskmmic.data.database.model
+
+import co.nimblehq.blisskmmic.domain.model.Token
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class TokenDatabaseModel(
+    @SerialName("access_token")
+    val accessToken: String,
+    @SerialName("token_type")
+    val tokenType: String,
+    @SerialName("expires_in")
+    val expiresIn: Int,
+    @SerialName("refresh_token")
+    val refreshToken: String,
+    @SerialName("created_at")
+    val createdAt: Int
```

Comment:
I think it would be better to make the attributes of this class nullable with the `?` symbol to avoid potential crashes in case some attributes can't be serialized properly (changes in attribute name, no attribute found, etc.) 🙏

Reply:
Since this will be returned in a flow, the result would be a flow error that should be handled with `catch` is it? Should we still nullable these properties and also introduce a custom error for it?

Reply:
`Can't be serialized properly` is also handled in `DataStoreImpl` 
```
settings
                .decodeValueOrNull(serializer, key)
                ?.let { emit(it) }
                ?: error(MR.strings.common_error.toString())
```

Reply:
> Since this will be returned in a flow, the result would be a flow error that should be handled with catch is it? Should we still nullable these properties and also introduce a custom error for it?

Well the question now is whether do we want to **throw an error** or **keep returning the data class with unparseable the fields as `null`** if such situation happens. 🤔 It might be better to not forcing all fields to have a value but yeah it is just my 2-cents 🙇‍♂️

Reply:
I see this Token object as a binary (exist or not) so I don't see the use case for having some of the property nullable. The concern will be useful when dealing with other data that's more of a UI data than Security data. 🙏